### ISSUE TITLE
Users or user groups access by folder

### ISSUE THREAD

**Issue by simonaubertbd** [RootDBApp/rootdb#11]
Hello,

I have noticed you can create folders and user groupes. However, I don't think you can set access rights to users by folders. Like "user group A can develop in folder B but can only consult in folder C and group D can only access to folder B".

It's a mandatory feature when you access several audiences.

Best regards,

Simon

**Comment by spo-ijaz** [comment]
Yes, you can restrict the access to a report on a list of users or user groups but we : 
* do not have user access restrictions on report's categories / directories ;
* and a user can only be a developer / viewer at the organization level. 

This task is linked to this one #7 - we need to think how we handle that.
